config/pmd/ruleset.xml

```diff
@@ -4,31 +4,40 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
 
-  <description>
-    HMCTS PMD rule set
-  </description>
+  <description>HMCTS PMD rule set</description>
```

Comment:
Is there a plan to update other java projects with this change?

Reply:
We should update pmd tool. Better rule tuning definitely will happen in bigger projects and those can be summarised over here